### PR TITLE
Changes to HP markers and miscellaneous additions to `random_messages`

### PR DESCRIPTION
**Pull request by src4026**
# Changes
- HP markers now append itemstrings.
- When HP is `0`, i.e., when a player places a marker upon death, is shows a different message: `"<player> died here"`
- Adds random messages about `/mhp`/HP markers, `/map` command, `mode:all` addition to the `/r` command.

### CODE REVIEW

**Comment by farooqkz** [comment]
@src4026 I am not against merging this one. But I believe it's a good idea to restrict each PR to a group of related changes.

File: mods/other/random_messages/init.lua [hunk 1]
```diff
@@ -75,6 +75,10 @@ function random_messages.read_messages()
 		"Use /mp <player> to send a marker to a specific teammate",
 		"To check someone's league in all modes, use /league <player>",
 		"Use /leagues to list leagues and the placement needed to get to them.",
+		"Use /mhp or left/right click while holding the zoom key (default: Z)"
+		.. "pointed towards your feet to place an HP marker.",
+		"Use /map to print the current map name and map author.",
```
Comment (farooqkz) [inline mods/other/random_messages/init.lua:80]:
```suggestion
		"Use /map to print the current map name and its author.",
```

**Comment by src4026** [comment]
As a side note, I have not tested the death message change feature because apparently in singleplayer CTF/or the one I have locally, there is no respawn screen as on the official server.

**Comment by LoneWolfHT** [comment]
> As a side note, I have not tested the death message change feature because apparently in singleplayer CTF/or the one I have locally, there is no respawn screen as on the official server.

Did you `/ctf_start` first?

File: mods/other/random_messages/init.lua [hunk 2]
```diff
@@ -75,6 +75,10 @@ function random_messages.read_messages()
 		"Use /mp <player> to send a marker to a specific teammate",
 		"To check someone's league in all modes, use /league <player>",
 		"Use /leagues to list leagues and the placement needed to get to them.",
+		"To place an HP marker run /mhp or left/right click while holding the zoom key (default: Z) "..
```
Comment (a-blob) [inline mods/other/random_messages/init.lua:78]:
```suggestion
		"To place an HP marker, run /mhp or left/right click while holding the zoom key (default: Z) "..
```

**Comment by a-blob** [comment]
> As a side note, I have not tested the death message change feature because apparently in singleplayer CTF/or the one I have locally, there is no respawn screen as on the official server.

Respawn delay only applies if the match has started.

**Comment by src4026** [comment]
Oh, that's why.

**Pull request merged**